docs/installation.md

```diff
@@ -122,6 +122,42 @@ Create a Service with type `LoadBalancer` using the appropriate manifest for you
    nslookup <dns-name>
    ```
 
+## Upgrading NGINX Gateway Fabric
+
+### Upgrade NGINX Gateway Fabric from Manifests
+
+1. Upgrade the Gateway Resources
+
+    Before you upgrade, ensure the Gateway API resources are the correct version as supported by the
+    NGINX Gateway Fabric - [see the Technical Specifications](/README.md#technical-specifications).
+
+    To upgrade the Gateway resources from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+
+    ```shell
+    kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.8.0/standard-install.yaml
+    ```
+
+1. Upgrade the NGINX Gateway Fabric CRDs
```

Comment:
Do we need to wait for this PR to be merged first https://github.com/nginxinc/nginx-gateway-fabric/pull/1092 ?

Reply:
I think the doc changes for #1092 will be a part of the 1.0 release, so we shouldn't wait for it to merge. This doc will need to be updated as part of the other doc updates for 1.0